roles/contiv_network/tasks/main.yml

```diff
@@ -107,6 +107,15 @@
   retries: 9
   delay: 10
 
+# get the netmaster status for debugging in case there are errors
+- name: get netmaster status
```

Comment:
how does it different from https://github.com/contiv/ansible/blob/master/roles/contiv_network/tasks/main.yml#L102